client/TracyProfiler.cpp

```diff
@@ -32,6 +32,14 @@
 #  include <sys/syscall.h>
 #endif
 
+#ifdef __ANDROID__
+#  include <sys/mman.h>
+#  include <cstdio>
+#  include <cstdint>
```

Comment:
Use C headers (e.g. `stdio.h`) and `std::`-less types.

Reply:
done